src/js/editor/editor.js

```diff
@@ -648,7 +622,7 @@ class Editor {
 
   insertSectionAtCursor(newSection) {
     let newRenderNode = this._renderTree.buildRenderNode(newSection);
-    let renderNodes = this.getRenderNodesWithCursor();
+    let renderNodes = this.cursor.activeSections.map(s => s.renderNode);
```

Comment:
`getRenderNodesWithCursor` returned render nodes for the markers- `activeSections` will return the sections?

Besides the confusion about that change, I'm unsure if it makes more sense to be talking about "what markers are selected" vs "what render nodes are selected". It is understandable that you can always find one from the other, but my leaning has been to make the render node the object interacted with by the editor as much as possible. This turns it around a little to be focused on the dom-ignorant post nodes.